models/discordactions.js

```diff
@@ -30,6 +30,42 @@ const createNewRole = async (roleData) => {
   }
 };
 
+const removeMemberGroup = async (roleId, discordId) => {
+  try {
+    const discordResponse = await removeRoleFromUser(roleId, discordId);
+    if (discordResponse) {
+      const backendResponse = await deleteRoleFromDatabase(roleId, discordId);
+      return backendResponse;
+    }
+  } catch (error) {
+    logger.error(`Error while removing role: ${error}`);
+    throw new Error(error);
+  }
+  return false;
+};
+
+const deleteRoleFromDatabase = async (roleId, discordId) => {
+  try {
+    const rolesToDeleteSnapshot = await memberRoleModel
+      .where("userid", "==", discordId)
+      .where("roleid", "==", roleId)
+      .limit(1)
```

Comment:
why was `limit(1)` used when there can be only 1 such document anyway

Reply:
issue fixed